.expeditor/config.yml

```diff
@@ -8,19 +8,18 @@ product_key:
 slack:
   notify_channel: chef-ws-notify
 
+docker_images:
+  - chefworkstation
```

Comment:
What is the reason for this over `chef-workstation`?

Reply:
it could go either way. Now is our chance

Reply:
I vote `chef-workstation` since it is easier to read

Reply:
Push away